css/at-rules/page.json

```
@@ -44,6 +44,41 @@
             "deprecated": false
           }
         },
+        "page": {
+          "__compat": {
+            "description": "<code>page</code> property",
+            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/@page/page",
```

```suggestion
            "mdn_url": "https://developer.mozilla.org/docs/Web/CSS/page",
```

this is a property, not a descriptor

---

this should not be under the css/at-rules/page.json, as it's not an at rule

---

Done